### PR TITLE
Remove older, slower Winston from detailed logging page

### PR DESCRIPTION
**Pull request by denisecase**
@kevynb , @goldbergyoni @josh-hemphill - updated the detailed logging page to remove Winston (no longer a recommended best practice) in response to #684 and #791 discussions

### CODE REVIEW

**Review by josh-hemphill — CHANGES_REQUESTED** [review]
I think having a code example about how to provide good context could be useful.

Also while we're here, do you mind adding the missing blockquote symbols before the blog quote list items?

**Comment by denisecase** [comment]
I agree - let's ask an experienced Pino user to add some examples.

File: sections/errorhandling/usematurelogger.md [hunk 1]
```diff
@@ -2,43 +2,19 @@
...
-const logger = new winston.Logger({
-  level: 'info',
-  transports: [
-    new (winston.transports.Console)()
-  ]
-});
-
-// custom code somewhere using the logger
-logger.log('info', 'Test Log Message with some parameter %s', 'some parameter', { anything: 'This is metadata' });
-```
```
Comment (kevynb) [inline sections/errorhandling/usematurelogger.md:23]: I think the code example looks like that given pino docs:

```javascript
const pino = require('pino');

// your centralized logger object
const logger = pino();

// custom code somewhere using the logger
logger.info({ anything: 'This is metadata' }, 'Test Log Message with some parameter %s', 'some parameter');
```

File: sections/errorhandling/usematurelogger.md [hunk 2]
```diff
@@ -2,43 +2,19 @@
...
 
-We all love console.log but obviously, a reputable and persistent logger like [Winston][winston] (highly popular) or [Pino][pino] (the new kid in town which is focused on performance) is mandatory for serious projects. A set of practices and tools will help to reason about errors much quicker – (1) log frequently using different levels (debug, info, error), (2) when logging, provide contextual information as JSON objects, see example below. (3) Watch and filter logs using a log querying API (built-in in most loggers) or a log viewer software. (4) Expose and curate log statement for the operation team using operational intelligence tools like Splunk.
+We all love console.log but obviously, a reputable and persistent logger like [Pino][pino] (a [newer option focused on performance][dropdiscussion]) is mandatory for serious projects. 
 
-[winston]: https://www.npmjs.com/package/winston
-[pino]: https://www.npmjs.com/package/pino
+A set of practices and tools will help to reason about errors much quicker.
+Recommendations include:
+
+1. Log frequently using different levels (debug, info, error).
```
Comment (kevynb) [inline sections/errorhandling/usematurelogger.md:10]: We try to keep the One paragraph Explainer a paragraph as much as possible :) Could you remove the line-breaks on line 6 and 7 at least?

File: sections/errorhandling/usematurelogger.md [hunk 3]
```diff
@@ -2,43 +2,19 @@
 
 ### One Paragraph Explainer
 
-We all love console.log but obviously, a reputable and persistent logger like [Winston][winston] (highly popular) or [Pino][pino] (the new kid in town which is focused on performance) is mandatory for serious projects. A set of practices and tools will help to reason about errors much quicker – (1) log frequently using different levels (debug, info, error), (2) when logging, provide contextual information as JSON objects, see example below. (3) Watch and filter logs using a log querying API (built-in in most loggers) or a log viewer software. (4) Expose and curate log statement for the operation team using operational intelligence tools like Splunk.
+We all love console.log but obviously, a reputable and persistent logger like [Pino][pino] (a [newer option focused on performance][dropdiscussion]) is mandatory for serious projects.
```
Comment (kevynb) [inline sections/errorhandling/usematurelogger.md:5]: I would remove `[dropdiscussion]` and add a line below that says something like: `If you are wondering why winston is not present here see #684 `

**Review by kevynb — COMMENTED** [review]
@denisecase thanks! I'm glad that you made the PR! 😃🎆

File: sections/errorhandling/usematurelogger.md (hunk 3 above)
Comment (denisecase) [reply sections/errorhandling/usematurelogger.md:5]: Agreed - adding 

For more information on why traditional favorites (e.g., Winston) may not be included in the current list of recommended best practices, please see #684.

File: sections/errorhandling/usematurelogger.md (hunk 2 above)
Comment (denisecase) [reply sections/errorhandling/usematurelogger.md:10]: Good idea. Please see updated version.

File: sections/errorhandling/usematurelogger.md (hunk 1 above)
Comment (denisecase) [reply sections/errorhandling/usematurelogger.md:23]: Perfect - thanks! Please see the updated version.

File: sections/errorhandling/usematurelogger.md (hunk 3 above)
Comment (kevynb) [reply sections/errorhandling/usematurelogger.md]: Looks good! From [what I see](https://github.com/denisecase/nodebestpractices/blob/master/sections/errorhandling/usematurelogger.md), It seems that github does not interpret the # issue number in markdown files. Would you mind transforming it into a link?
After that it's good to merge for me :)

Comment (denisecase) [reply sections/errorhandling/usematurelogger.md]: Thanks @kevynb - I've enabled the link in the text.. it's relative, so it won't find 684 while over in my fork, but I think it'll work once brought back home.

Comment (kevynb) [reply sections/errorhandling/usematurelogger.md]: Ah I understand. Thanks for explaining to me!

**Review by kevynb — APPROVED** [review]

**Comment by kevynb** [comment]
@all-contributors please add @denisecase for content

**Pull request merged**